### PR TITLE
APIGOV-14293 Update Add Environment Documentation

### PR DESCRIPTION
**Pull request by rsavajiyani**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

We recently refactored the Add Environment flow. These docs have been updated to update the new user experience.

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is clear and complete
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://techweb.axway.com/confluence/display/RDAPI/Best+practices+for+developer+documentation)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue

### CODE REVIEW

**Comment by rsavajiyani** [comment]
@robertengaxway , @markbice I've updated the Add Environment documentation to reflect the new user experience. I chose to consolidate both the API Gateway V7 flow and the Kubernetes flow into the same file, so that this documentation section could serve as the single source of truth for both. Please let me know if you think I should split them, keeping one page for the hybrid documentation, and another for v7 documentation

@tsjohns9 let me know if you see anything needing revising.

**Comment by robertengaxway** [comment]
Both docs updates look good to me.   My only suggestion explain what "key:value pair attributes" are and to give an example of the attributes for new users.   Its in the section, "Add your API Gateway V7 Environment"

**Comment by rsavajiyani** [comment]
> Both docs updates look good to me. My only suggestion explain what "key:value pair attributes" are and to give an example of the attributes for new users. Its in the section, "Add your API Gateway V7 Environment"

Thanks @robertengaxway for your feedback, I've added an example key:value pair to make it more clear.

File: content/en/docs/central/add_env.md [hunk 1]
```diff
@@ -49,7 +91,7 @@ ls
 hybridOverride.yaml  istioOverride.yaml
 ```
 
-## Generate a key pair and secret for the domain edge gateway
+#### Generate a key pair and secret for the domain edge gateway
```
Comment (andreamussap) [inline content/en/docs/central/add_env.md]: @rsavajiyani I see that you've changed some headings from H2 to H4 under **Add your Kubernetes Environment**, so I just wanted to double check that all these headings that you changed to H4 are child of **Add your Kubernetes Environment** section ??

**Comment by andreamussap** [comment]
@robertengaxway my review is done. Please let me know if I can approve and merge it. Thanks.

https://deploy-preview-514--axway-open-docs.netlify.com/docs/central/add_env/

**Comment by andreamussap** [comment]
### Thank you
We really appreciate your contribution and thank you for helping us to improve this documentation!
 
### Where to find the updated documentation
An Axway technical writer has reviewed and merged your PR, which is now available on the following sites:
Netlify (staging): https://axway-open-docs.netlify.com/docs/
Zoomin (production): https://docs.axway.com/bundle/axway-open-docs/
 
### Send us your feedback
If you have a moment, please complete the following 2 minute survey to give us your feedback on this process and help us make it even better:
https://www.surveymonkey.com/r/PJ5HQFD

**Pull request merged**